frappe/public/js/drawing/whiteboard_editor.bundle.jsx

```diff
@@ -0,0 +1,26 @@
+import * as React from "react";
+import { App } from "./App";
+import { createRoot } from "react-dom/client";
+
+class WhiteboardEditor {
```

Comment:
Ummm... do we need to add react dependency for 26 line code? We already have Vue :sweat_smile:

Reply:
ah Exclaidraw embed only works with React?

Reply:
Yup 😅

Reply:
Also, won't load any JS until the user clicks "Open Editor".